charts/bivac/values.yaml

```diff
@@ -0,0 +1,50 @@
+images:
+  bivac:
+    repository: camptocamp/bivac
+    tag: 2.0.0
+    pullPolicy: IfNotPresent
+
+config:
```

Comment:
Maybe we should just expose a `bivac.env: {}` parameter to have maximum flexibility.